docs/guides/guides.md

```diff
@@ -61,3 +61,7 @@ Specifically, this guide will walk through the implementation of the following b
  - Handling relationships between model types.
  - Setup node services and server-side rendering on the same process.
  - How to turn off parts of the app that should not be server-side rendered.
+
+## [Contributing](./contributing.html)
```

Comment:
should be able to do `[contributing Contributing]` instead, using the doc name prevents you from having to hardcode the link like that.